test/spec/modules/parrableIdSystem_spec.js

```diff
@@ -249,6 +249,20 @@ describe('Parrable ID System', function() {
       expect(resolvedOptions.called).to.equal(true);
     });
 
+    it('permits an impression from a lower cased allowed timezone', function() {
+      const allowedZone = 'america/new_york';
+      const resolvedOptions = sinon.stub().returns({ timeZone: allowedZone });
+      Intl.DateTimeFormat.returns({ resolvedOptions });
+
+      expect(parrableIdSubmodule.getId({ params: {
+        partner: 'prebid-test',
+        timezoneFilter: {
+          allowedZones: [ allowedZone ]
+        }
```

Comment:
this is not actually testing the behavior because you are stubbing the Intl api with the lowercase version and also supplying it in the config.
You should return `America/New_York` from the resolvedOptions stub and supply the lowercase version to the allowedZones config.

The same changes should apply to the blocked timezone test as well.